crates/node/src/network_server/metrics.rs

```diff
@@ -277,11 +275,13 @@ pub async fn render_metrics(State(state): State<NodeCtrlHandlerState>) -> String
 
         // Properties (Gauges)
         // For properties, we need to get them for each column family.
+        let mut cf_labels = Vec::with_capacity(labels.len() + 1);
+        labels.clone_into(&mut cf_labels);
+        let last_cf_labels_idx = labels.len();
+        cf_labels.push(String::from("cf=\"\""));
         for cf in &db.cfs() {
             let sanitized_cf_name = formatting::sanitize_label_value(cf);
-            let mut cf_labels = Vec::with_capacity(labels.len() + 1);
-            labels.clone_into(&mut cf_labels);
-            cf_labels.push(format!("cf=\"{sanitized_cf_name}\""));
+            cf_labels[last_cf_labels_idx] = format!("cf=\"{sanitized_cf_name}\"");
```

Comment:
Could we reuse `labels` instead of creating a new `cf_labels` here? It should work if we create initially a vec with 2 additional entries.

Reply:
ok, i will work on it.